{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -28,14 +28,15 @@ click = ">=8.0.1"
 Pygments = ">=2.10.0"
 black = ">=21.10b0"
 coverage = {extras = ["toml"], version = ">=6.2"}
-darglint = ">=1.8.1"
+darglint = {version = ">=1.8.1", python = "<4"}
+urllib3 = {version = ">=1.26.8", python = "<4"}
```

Comment:
should urllib be here?

Reply:
@paw-lu When I tested I saw that if you don't set the version of urllib3 then a version of urllib3 was downloaded that had a bug, and the library that urllib3 used was darglint so I put it there.

Reply:
urllib3 should not be here. It requires Python < 4, and it's an indirect dependency via safety and sphinx. W could make this work by adding the environment marker to safety and sphinx as well. However, I'm now considering whether we should add the Python cap back. I'll open an issue shortly so we can discuss the pros and cons of this move.

Reply:
See #1154